src/Microsoft.ML.Transforms/Text/WrappedTextTransformers.cs

```diff
@@ -39,13 +39,13 @@ public WordTokenizer(IHostEnvironment env, string inputColumn, string outputColu
         /// </summary>
         /// <param name="env">The environment.</param>
         /// <param name="columns">Pairs of columns to run the tokenization on.</param>
-        /// <param name="separators">The separators to use (comma separated).</param>
-        public WordTokenizer(IHostEnvironment env, (string input, string output)[] columns, string separators = "space")
+        /// <param name="separators">The separators to use (uses space character by default).</param>
```

Comment:
> by default) [](start = 80, length = 12)

if its value is set to <value>null</value>

Reply:
You cannot have a default value of new[] { ' ' } in the constructor, so I use null instead and have logic that takes care of that. It will switch null for the space character. 

---
In reply to: [222368628](https://github.com/dotnet/machinelearning/pull/1089#discussion_r222368628) [](ancestors = 222368628)